tensorflow/python/ops/data_flow_ops.py

```diff
@@ -2088,6 +2088,7 @@ class RecordInput(object):
   def __init__(self,
                file_pattern,
                batch_size=1,
+               minibatches=None,
```

Comment:
I think it's best to add new optional parameters to the end of the list, otherwise you break callers not using named parameters.